deploy/clowdapp.yaml

```diff
@@ -899,10 +899,10 @@ parameters:
     value: pulp-stage-default-domain-s3
   - name: PULP_AUTHENTICATION_BACKENDS
     description: The AUTHENTICATION_BACKENDS to be used by Pulp.
-    value: "@merge django.contrib.auth.backends.RemoteUserBackend" 
+    value: "@merge django.contrib.auth.backends.RemoteUserBackend"
   - name: PULP_REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES
     description: The defaul authentication classes for Pulp.
-    value: "@merge pulpcore.app.authentication.JSONHeaderRemoteAuthentication"
+    value: "['pulpcore.app.authentication.JSONHeaderRemoteAuthentication','rest_framework.authentication.BasicAuthentication','rest_framework.authentication.SessionAuthentication']"
```

Comment:
Is session auth (cookie auth) used at all for services? I would recommend putting it before BasicAuth if it is.

Reply:
I believe it's gonna be used with the django-admin that we enabled on services. Good catch.

Reply:
let's also add pulp_service.app.authentication.RHServiceAccountCertAuthentication to this list here. and if you make it first in the list, it will return the proper 401 (with the other change you already have in this PR).